test/mysql_handler/test

```diff
@@ -1,20 +1,55 @@
 #!/bin/bash -ex
 
 local_test=false
-while getopts :l opt; do
+while getopts ":lb" opt; do
     case $opt in
         l) local_test=true ;;
+				b) benchmark=true ;;
        \?) echo "Unknown option -$OPTARG"; exit 1;;
     esac
 done
 
 docker_args=""
 if $local_test; then
-  docker_args="-v $(cd ../..; pwd):/go/src/github.com/cyberark/secretless-broker"
+  docker_args="-v $(cd ../..; pwd):/secretless"
 fi
 
-docker-compose run \
-  --rm \
-  --no-deps \
-  $docker_args \
-  test
+if $benchmark; then
+  # first set MYSQL_ADDRESS and SECRETLESS_ADDRESS
+  mysql_cid=$(docker-compose ps -q mysql)
+  secretless_cid=$(docker-compose ps -q secretless)
+  mysql_ip=$(docker inspect \
+    -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $mysql_cid)
+  secretless_ip=$(docker inspect \
+    -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $secretless_cid)
+  export MYSQL_ADDRESS=$mysql_ip:3306
+  export SECRETLESS_ADDRESS=$secretless_ip:15432
```

Comment:
This comment addresses lines 19 to 26.

All things being default,`docker-compose` services are made available to containers in other services of the same docker-compose namespace via their service names. This means you can simplify the above to:
```
export MYSQL_ADDRESS=mysql:3306
export SECRETLESS_ADDRESS=secretless:3306
```